framework/audio/engine/audioenginemodule.cpp

```diff
@@ -64,10 +73,23 @@ void AudioEngineModule::registerUiTypes()
 
 void AudioEngineModule::onInit()
 {
-    audioEngine->init();
+    _audioEngine->init();
+
+    _audioChannelInvoker = std::make_shared<mu::framework::Invoker>();
+
+    _audioChannelInvoker->onInvoked([]() {
```

Comment:
Before I did the invoker, notifications and channels could not send messages between threads.
Now, yes, I can replace invoker with notification.
But the invoker has its own essence, his job is to perform a given function in the invoker's thread. 
In this case, the task is to execute the specified function in the invoker thread.
It can be further developed to pass arguments.